src/common/Widget.js

```diff
@@ -122,7 +122,7 @@
     };
 
     // Serialization  ---
-    Widget.prototype.require = require || function (paths, cb) {
+    Widget.prototype.require = window.require || function (paths, cb) {
```

Comment:
"window.require" should be passed into the "factory" function within the UMD block.